sdcm/tester.py

```diff
@@ -1150,7 +1150,7 @@ def cql_connection(self, node, keyspace=None, user=None,  # pylint: disable=too-
                        password=None, compression=True, protocol_version=None,
                        port=None, ssl_opts=None, connect_timeout=100, verbose=True):
         # TODO: ask Bentsi why it was reverted (PR #1236)
```

Comment:
@bentsi, why it was reverted? : )

Reply:
we can remove this comment since we are running the code from the PR #1236 :)

Reply:
removed )